pkg/policy/validate.go

```diff
@@ -140,6 +188,12 @@ func Validate(policy kyvernov1.PolicyInterface, client dclient.Interface, mock b
 		if err := validateJSONPatchPathForForwardSlash(rule.Mutation.PatchesJSON6902); err != nil {
 			return nil, fmt.Errorf("path must begin with a forward slash: spec.rules[%d]: %s", i, err)
 		}
+		if err := validateJSONPatchKind(rule.Mutation.PatchesJSON6902); err != nil {
+			return nil, fmt.Errorf("Unexpected kind: spec.rules[%d]: %s", i, err)
+		}
+		if err := validateJSONPatchValue(rule.Mutation.PatchesJSON6902); err != nil {
```

Comment:
Can we combine these two checks? It could save us sometime decoding the patches.